ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -66,11 +67,17 @@ const GCNV_TRACK_OPTIONS = {
   onlyHandleClicksForHighlightedSamples: true,
 }
 
+const ANNOTATION_TRACK_OPTIONS = {
+  format: 'gtf',
+  height: 100,
+}
+
 const TRACK_OPTIONS = {
   [ALIGNMENT_TYPE]: ALIGNMENT_TRACK_OPTIONS,
   [COVERAGE_TYPE]: COVERAGE_TRACK_OPTIONS,
   [JUNCTION_TYPE]: JUNCTION_TRACK_OPTIONS,
   [GCNV_TYPE]: GCNV_TRACK_OPTIONS,
+  [ANNOTATION_TYPE]: ANNOTATION_TRACK_OPTIONS,
```

Comment:
we do not support adding these tracks in seqr and that was not part of any of the tickets for this so I don't know why you added this, but please take it out as we do not have any intention of supporting this

Reply:
We have a SegDups track for mappability track types which is an "annotation" type.

Reply:
thats a reference track so you should hard code the constant with the appropriate settings, not  change the track options for actual tracks

Reply:
I could hard code the track options for this reference track. But how about the other reference tracks which have types like COVERAGE_TYPE and JUCTION_TYPE, should I use the same track options as the actual tracks with the same types?

Reply:
I think the reference tracks need to be as clear as possible, which with all the current mapping they really aren't. Maybe do something like this to remove some duplication but keep things explicit
```
const GTEX_TRACKS = [...].map(track => ({ ...GTEX_PROPS, ...track })
const MAPPING_TRACKS = [...].map(track => ({ ...MAPPING_PROPS, ...track })
```

Reply:
Good idea. I updated the code. Let me know if any further issues.